README.md

```diff
@@ -18,6 +18,7 @@
 5. have bot listing requirements (such as no NSFW outside NSFW channels, respect rate-limits, don't respond to other bots, etc.)
 6. blur NSFW profile pictures.
 7. have a ToS and Privacy page.
+8. not be a copy of other's bot lists (open sourced lists, including lists that are edited or modified from other's lists).
```

Comment:
Please move to the _should_ section

Reply:
I cant see "should". But I think if you guys allow lists that are forked off, there will be like tens of lists getting added to botblock everyday. Which might not look good

Reply:
BotBlock has always permitted lists using an open-source base as long as they are compliant with the license, there is no reason not to. The _should_ section is in this file, below the first list.

Reply:
also there might be dmca complaints, and many bot devs might scream to botblock about why are they approving fake lists that are forked off. Also see this, I dont see the point of listing them for the work that they didnt do.

Reply:
In such a case can Matt tell them that we only list the sites and don't own them.
It's not the job of BotBlock to play Internet-police and make sure everyone follows the TOS, Privacy Policy, Guidelines and licenses a template may have.

Reply:
I would suggest have a voting on the server itself, and I bet you will see everyone is against forked lists. The only ones who will be supporting will be a few, and they would mostly include people who own forked off list.

Reply:
Could you just stop already?
Matt already gave his clear opinions on this matter and you trying to constantly argue against this doesn't serve any benefit nor purpose and only makes you look like a fool.

You should stop comparing BotBlock, a site that lists bot lists, with bot lists. Those are two separate concepts.

And as it was mentioned many, MANY times before does BotBlock not judge a site by wether it uses a template or is original. If the site works, is fairly secure (Doesn't have sensitive data public) and has reliable uptime, then it's fine. It's not our job here to judge if a site copying a template is automatically bad. Otherwise, every site using HTML5Up themes or one of the various GitHub Pages themes would by your logic be shit.

Some people may not have the knowledge, time and/or money to get a proper, own theme setup, so a template is the best option for them.